### DISCUSSION TITLE
Environment variables support

### DISCUSSION THREAD

**Discussion by jollymover** [mdhowle/ansible-ad-inventory#17] [Q&A] [answered]
Hi Matthew!

Thank you for the plugin. It work like charm.

Any options to use an environment variable instead of password field in ad.yml?

**Comment by mdhowle** [comment]
Sure, I'll add that in next week.

**Reply by mdhowle** [reply, marked as answer]
I haven't tested, but I push a change to the `environ` branch. 
You can set username and password with `ANSIBLE_AD_INVENTORY_USERNAME`  and `ANSIBLE_AD_INVENTORY_PASSWORD` respectively.
